src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -235,7 +237,7 @@ public synchronized void start() {
   public synchronized void stop() {
     if (_running.compareAndSet(true, false)) {
       try {
-        _baseConsumer.close();
+        _running.set(false);
```

Comment:
This won't do anything since _running.compareAndSet would have already set this to false.